lib/dal/tests/integration_test/secret/with_actions.rs

```diff
@@ -63,24 +64,13 @@ async fn create_action_using_secret(ctx: &mut DalContext, nw: &WorkspaceSignup)
     .expect("could not find prop by path");
 
     // Connect the two components to propagate the secret value and commit.
-    let source_output_socket = OutputSocket::find_with_name(ctx, "dummy", source_schema_variant_id)
-        .await
-        .expect("could not perform find with name")
-        .expect("output socket not found by name");
-    let destination_input_socket =
-        InputSocket::find_with_name(ctx, "dummy", destination_schema_variant_id)
-            .await
-            .expect("could not perform find with name")
-            .expect("input socket not found by name");
-    Component::connect_for_tests(
+    value::subscribe(
```

Comment:
This tests secrets, not socket connections, so it becomes a subscription instead.